docs/_docs/03-ug-mutable-dataset.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Mutable dataset"
+permalink: /docs/ug-mutable-dataset/
+excerpt: "Mutable dataset guide"
+last_modified_at: 2020-10-14
+toc: false
+classes: wide
+---
+
+This guide helps you to utilize indexes on mutable dataset.
+
+## What does mutable mean?
+Hyperspace can build indexes based on the source files in the given logical plan.
+Since Hyperspace v0.3, we support file-level "mutable" dataset which means appended files and/or
+deleted files under the root paths of the given logical plan.
```

Comment:
Again I think we should not talk about "logical plan" here and simply talk about "files".